app/views/students/_student_info.html.erb

```diff
@@ -6,167 +6,161 @@
       </div>
     </div>
     <table id="datatable">
-      <tbody>   
+      <tbody>
         <tr>
           <td style="padding-left: 40px;"><p style="font-size: 18px">Nombre:</p></td>
-          <td><p style="font-size: 18px"><%= user.student.name %></p></td>
-        </tr>   
+          <td><p style="font-size: 18px"><%= @student.name %></p></td>
```

Comment:
Use the parameters you sent, don't use the instance variables since it's harder to reuse code.

Reply:
Ditto.